packages/cozy-sharing/README.md

```
@@ -1,83 +1,289 @@
+| Org shared drive | `isOrgSharedDrive` = `sharing.drive === true && sharing.org_drive === true` | `state.js:324` |
+| `canLeave` | `false` for org shared drives | `state.js:335` |
+| `canReshare` | `!org_drive && !read_only` for drives; `open_sharing && !read_only` for folders | `state.js:342` |
+| Federated folder modal | Enabled by `drive.federated-shared-folder.enabled` flag | `ShareModal.jsx:39` |
+| Email sharing in federated folder | Disabled for files inside a federated shared folder (has `driveId` but is not the root) | `FederatedFolderModal.jsx:122-126` |
+| Link sharing in federated folder | Uses `getFederatedShareLink` which resolves the owner's instance URL | `FederatedFolderModal.jsx:108-110` |
+
+### Share management
+
+| Condition | Rule | Source |
+|-----------|------|--------|
+| Editable modal | `isEditable = !byDocId[doc] || isOwner(doc) || canReshare(doc)` | `ShareModal.jsx:35-36` |
```

markdownlint-cli2 (0.23.1)

[warning] Table column count
Expected: 3; Actual: 7; Too many cells, extra data will be missing

(MD056, table-column-count)